wasm-rpc/Cargo.toml

```diff
@@ -15,13 +15,14 @@ crate-type = ["cdylib", "rlib"]
 harness = false
 
 [dependencies]
-wit-bindgen-rt = { version = "=0.36.0", features = ["bitflags"] }
+wit-bindgen-rt = { version = "=0.39.0", features = ["bitflags"] }
```

Comment:
Let's keep using the same version that is in our current recommended (= depended in the stubgen and mentioned in our docs) cargo-component, 0.20, which depends on wit-bindgen 0.36